modules/ROOT/pages/cassandra/cassandra-studio.adoc

```diff
@@ -0,0 +1,112 @@
+= Studio - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Install Cassandra
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
+. Click Login in Anypoint Exchange and supply your Anypoint Platform username and password.
+. Search for the Cassandra connector and click Install.
+. Follow the prompts to install the connector.
+
+== Configure Cassandra
+. Drag and drop the operation connector to the Studio canvas.
++
+[%header%autowidth.spread]
+|===
+|Operation |Description
+|Add New Column a|Input for this operation is:
+
+* Parameter: A table name
+* Optional keyspace name
+* Payload: an AlterColumnInput POJO
++
+The POJO has the following fields:
++
+** Name of the column to be created
+** The data type for the new column
+
+|Change Column Type a|  Input for this operation is:
+
+** Parameter: A table name
+** Optional keyspace name
+** Payload: an AlterColumnInput POJO
+|Create Keyspace a| Input for this operation using SimpleStrategy is:
+
+* Keyspace name
+* Replication factor
```

Comment:
What's this? Where does the customer find it?

Reply:
Good question, I will have to investigate this on my second pass through the connector after putting them all in the templates.